maintainers/scripts/update-python-libraries

```diff
@@ -60,10 +64,19 @@ def _get_values(attribute, text):
 
     :returns: List of matches.
     """
+    # Find occurences of 'attribute = "string";'
```

Comment:
Why are the following changes needed? Could you update the docstring to describe what this function does.